storageimg_kvm.py

```diff
@@ -1,333 +0,0 @@
-"""
```

Comment:
Instead of deleting this file, should it be moved into the `cloudcix_primitives` path so work done to date is not lost? For example, the contract params, messages and payloads for each verb.

Reply:
I thought we dropped storageimg_kvm primitive because the cloudinit_kvm creates it as it is a primary storage a property of VM

Reply:
Thank you, I forgot that!